craft_parts/plugins/python_plugin.py

```diff
@@ -73,3 +77,38 @@ def _get_package_install_commands(self) -> list[str]:
         )
 
         return commands
+
+    @override
+    def get_files(self) -> PackageFiles:
+        # https://packaging.python.org/en/latest/specifications/binary-distribution-format/
+        # Could also add the pkginfo library for this
+
+        venvdir = self._get_venv_directory()
+        python_path = venvdir / "bin/python"
+        python_version = python_path.resolve().name
+        site_pkgs_dir = venvdir / "lib" / python_version / "site-packages"
+
+        ret = {}
+        for pkg_dir in site_pkgs_dir.iterdir():
+            # We only care about the metadata dirs
+            if not pkg_dir.name.endswith(".dist-info"):
```

Comment:
how about using `.glob("*.dist-info")` instead of `.iterdir()`

Reply:
https://github.com/canonical/craft-parts/pull/943/commits/6a8e8ce17a4a15f57e799bc9b815854bf65c4b24